Common/src/Pollster/TaskHandler.cs

```diff
@@ -589,33 +590,19 @@ await agent_.FinalizeTaskCreation(CancellationToken.None)
     logger_.LogDebug("End Task Processing");
   }
 
-  private async Task HandleErrorAsync(Exception         e,
-                                      TaskData          taskData,
-                                      CancellationToken cancellationToken)
+  private async Task HandleErrorRequeueAsync(Exception         e,
```

Comment:
I think it is a good idea to have 2 separate functions for those tasks, but the implementation is very much similar, so maybe they could call a single function, no?

Reply:
The "unified" function would be my first commit. It increase the complexity of the code too much.

Reply:
Just to be clear, my proposal was to have two functions when calling, but those two functions call a single one. From what I can see, the only real difference between the two is the status of the message, and if the task is actually resubmitted or not. Those can be taken in parameter of the helper function.

All in all, I don't mind the current implementation.